src/acme/Containers.h

```diff
@@ -53,4 +53,11 @@ InputIterator closest(InputIterator first, InputIterator last, ValueType value)
 	});
 }
 
+template <typename T>
+T& append(T& destination, const T& elements)
```

Comment:
I guess I would lean more to T&& destination here

Reply:
sure need some extra overrides here, now I didn't needed "move" append but just append with copy